MonoGame.Framework/Graphics/SpriteBatchItem.cs

```diff
@@ -2,10 +2,11 @@
 
 namespace Microsoft.Xna.Framework.Graphics
 {
-	internal class SpriteBatchItem
+    internal class SpriteBatchItem : IComparable<SpriteBatchItem>
 	{
 		public Texture2D Texture;
 		public float Depth;
```

Comment:
Do we actually use `Depth` anywhere other than in the `Set` methods below?  Can we just pass it into the vertex setup methods and not store it?